config/environments/production.rb

```diff
@@ -56,7 +56,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
```

Comment:
Is this comment still accurate? Seems like it should say something more like "make lookups for any locale fall back to 'en'"?

Reply:
No. Addressed in https://github.com/tootsuite/mastodon/pull/6843/commits/7820893aca79826b19c51da8dc834b13278db0b7. Thanks for pointing out this.